docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx

```diff
@@ -226,6 +249,7 @@ struct FloatingParticipantView: View {
 
     var participant: CallParticipant?
     var size: CGSize = .init(width: 120, height: 120)
+    var onChangeTrackVisibility: (CallParticipant?, Bool) -> Void
```

Comment:
We don't need this here, right? (the local video doesn't go out of screen while scrolling)

Reply:
👍